src/Utility/Legendre.C

```diff
@@ -35,7 +35,7 @@ bool Legendre::GL (RealArray& weights, RealArray& points, int n)
     A(n,n-1) = Real(n-1)/Real(2*n-1);
   }
 
-  RealArray eig_complex, evalpoints;
+  RealArray eig_complex(n), evalpoints(n);
```

Comment:
why? this resize is performed inside `solveEigNon()` afics..

Reply:
As the links in commit message shows, it triggers in tests.

Reply:
what links? I don't see them. But anyway I let this pass..

Reply:
I removed them for final merge, they were in the original commit messages before I started preparing for merge. http://afem:8080/job/IFEM-static-analysis/302/testReport/junit/(root)/debug-iter/TestLegendre_GL/

Reply:
OK, so yet another case of dereferencing a pointer to an empty array. This could then also more elegantly been resolved by using `r_val.data()`  and `c_val.data()` in the `dgeev_()` in `DenseMatrix::solveEigNon()`.

Reply:
it would not solve the underlying problem which is that nothing is resizing those vectors. This is a logic error.

Reply:
Really? Check again. The first call to `dgeev_()` (with `Lwork=-1`)  is only for obtaining the length of the work array needed. It will _not_ access the `val` and `vec` arguments (we could as well sent in the `nullptr` here instead). Then they are resized before the second call which actually does the work. Many of the LAPACK-subroutines work like this.

Reply:
right. in any case i see no problem with resizing up front.

Reply:
It is not a problem, just unneccessary